tests/Tokenizer/Analyzer/NamespaceUsesAnalyzerTest.php

```diff
@@ -111,6 +111,21 @@ public function provideNamespaceUsesCases()
                     21
                 ),
             ], [1, 12]],
+
+            // TODO: How to support these:
```

Comment:
These TODOs are inheritence of the NoUnusedImportsFixer. If it's OK to you, I would leave them as-is for now and add support in another PR.